src/utils/HTTPTransport.ts

```diff
@@ -59,10 +59,11 @@ class HTTPTransport {
             });
 
             xhr.onload = function () {
+                // @ts-ignore
```

Comment:
Попробуй вместо ts-ignore пользоваться инструментами TS, например приведение типов 
```suggestion
               resolve(xhr as Response);
```